src/applications/appeals/995/sass/995-supplemental-claim.scss

```diff
@@ -52,7 +52,7 @@ ul.issues,
 }
 
 article[data-location="issue-summary"] {
-  font-size: 1.5rem;
+  font-size: 0.9375rem;
   font-weight: 400;
```

Comment:
I don't remember why this was added in the first place, but it's breaking the page layout - I think we can just remove the font-size completely

| | |
|----|----|
| With font size of `0.9375` | <img width="816" alt="Screenshot 2024-04-25 at 1 34 37 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/cea2ded4-21d0-4c97-aadc-29d5fa229f0c"> |
| Without font size | <img width="773" alt="Screenshot 2024-04-25 at 1 34 53 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/136959/f05d2f54-1679-4826-842e-b6f28f15137b"> |

Reply:
Thanks for finding that! I'll remove that style. 👍